src/modules/InviteUser/router/index.js

```diff
@@ -0,0 +1,15 @@
+
+
+const InviteUser = ()=> import("../components/InviteUser.vue");
```

Comment:
Why you are importing from the components directory?

Reply:
update: removed the import